pkg/apiserver/admission/chain_builder.go

```diff
@@ -208,7 +208,7 @@ func NewAdmissionChains(
 		admissionPluginConfigFilename = tempFile.Name()
 	}
 
-	admissionPluginNames := CombinedAdmissionControlPlugins
+	admissionPluginNames := openshiftAdmissionControlPlugins
```

Comment:
make an issue indicating that we need to fix this entire method and assign it to me.

Reply:
Done https://github.com/openshift/origin/issues/21235